### DISCUSSION TITLE
Bulk delete/add to shelf

### DISCUSSION THREAD

**Discussion by Korina34** [bookwyrm-social/bookwyrm#2304] [Ideas]
Will there be an option to bulk delete or add to shelves?

**Comment by mouse-reeve** [comment]
Bulk delete is a feature I'd very much like to add, you can follow the issue for it if you'd like to get updates: https://github.com/bookwyrm-social/bookwyrm/issues/1763. Bulk add is a much more complicated proposition, and it's not currently planned.
